app/src/main/java/com/claudiumilea/newsreader/model/ArticleItemViewModel.java

```diff
@@ -0,0 +1,23 @@
+package com.claudiumilea.newsreader.model;
+
+import androidx.databinding.ObservableField;
+import androidx.lifecycle.ViewModel;
+
+public class ArticleItemViewModel extends ViewModel {
+    public final ObservableField<String> content, title;
```

Comment:
let's not define multiple members on the same line, not a very common code style for object oriented programming.